front/lib/swr/swr.ts

```diff
@@ -129,6 +131,20 @@ const resHandler = async (res: Response) => {
 
 export const fetcher = async (...args: Parameters<typeof fetch>) => {
   const [url, config] = args;
+
+  if (USE_PUBLIC_API && url.toString().startsWith("/api")) {
+    const transformedUrl = url.toString().replace("/api", USE_PUBLIC_API);
+
+    const res = await fetch(transformedUrl, {
+      ...config,
+      headers: {
+        Authorization: "Bearer sk-164b21d76cc2c7bf288b836d43ee77fd",
```

Comment:
😱

Reply:
Hopefully this is a local token, even though, let's try to avoid those mistakes to commit credentials in PR!

Reply:
Still it's not a good practice.